### PR TITLE
create station unify the components

### PR DESCRIPTION
**Pull request by SvetaMemphis**

### CODE REVIEW

File: ui_src/src/components/createStationForm/index.js [hunk 1]
```diff
@@ -62,70 +65,74 @@ const storageOptions = [
...
     const [creationForm] = Form.useForm();
     const [allowEdit, setAllowEdit] = useState(true);
+    const [actualPods, setActualPods] = useState(null);
+    const [retentionType, setRetentionType] = useState(retanionOptions[0].value);
+    const [storageType, setStorageType] = useState(storageOptions[0].value);
 
     useEffect(() => {
-        checkState();
-        createStationFormRef.current = onFinish;
+        getOverviewData();
```
Comment (avrhamNeeman) [inline ui_src/src/components/createStationForm/index.js:77]: Call it just if you are in get-started mode.

File: ui_src/src/components/createStationForm/index.js [hunk 2]
```diff
@@ -62,70 +65,74 @@ const storageOptions = [
...
 
-    const checkState = () => {
-        if (getStartedState?.formFieldsCreateStation?.name) {
-            setAllowEdit(false);
-            getStartedDispatch({ type: 'SET_NEXT_DISABLE', payload: false });
-        } else {
-            setAllowEdit(true);
+    useEffect(() => {
+        createStationFormRef.current = onFinish;
+    }, [createStationFormRef, getStartedStateRef]);
```
Comment (avrhamNeeman) [inline ui_src/src/components/createStationForm/index.js:84]: Why do you need these arguments and not always when component DidMount?

File: ui_src/src/components/modal/index.js [hunk 3]
```diff
@@ -27,7 +27,7 @@ import { Modal } from 'antd';
 import Button from '../button';
 
 const TransitionsModal = (props) => {
-    const { height, width, rBtnText, lBtnText, rBtnDisabled, lBtnDisabled, header, isLoading, open = false, displayButtons = true } = props;
+    const { height, width, rBtnText, lBtnText, rBtnDisabled, lBtnDisabled, header, isLoading, headerImage, subTitle, open = false, displayButtons = true } = props;
```
Comment (avrhamNeeman) [inline ui_src/src/components/modal/index.js:30]: You don't need a headerImage to display the image, you can send  as a header prop, and that will work.

File: ui_src/src/components/createStationForm/index.js (hunk 1 above)
Comment (SvetaMemphis) [reply ui_src/src/components/createStationForm/index.js:77]: in getOverviewData I receive the number of pods in order to get the maximal number of replicas. I need it in both modes.

File: ui_src/src/components/createStationForm/index.js (hunk 2 above)
Comment (SvetaMemphis) [reply ui_src/src/components/createStationForm/index.js:84]: You right. Fixed it.

File: ui_src/src/components/modal/index.js (hunk 3 above)
Comment (SvetaMemphis) [reply ui_src/src/components/modal/index.js]: Fixed

**Pull request merged**